qm-plugins/qm-cron/class-qm-cron-output-html.php

```diff
@@ -0,0 +1,333 @@
+<?php
+
+class QM_Cron_Output extends \QM_Output_Html {
+
+	private $doing_cron = false;
+
+	public function __construct( \QM_Collector $collector ) {
+		parent::__construct( $collector );
+
+		add_filter( 'qm/output/menus', array( $this, 'admin_menu' ), 110 );
+	}
+
+	public function admin_menu( array $menu ) {
+		$menu[] = $this->menu( array(
+			'id'    => 'qm-cron',
+			'href'  => '#qm-cron',
+			'title' => esc_html__( 'Cron', 'query-monitor' ),
+		));
+
+		return $menu;
+	}
+
+	public function output() {
+		$data             = $this->collector->get_data();
+		$this->doing_cron = $data['doing_cron'];
+		?>
+		<div class="qm qm-non-tabular" id="<?php echo esc_attr( $this->collector->id ); ?>">
+		<table id="qm-cron-stats" style="width: 100%">
+			<thead>
+				<tr>
+					<th><h3><?php esc_html_e( 'Total Events', 'query-monitor' ); ?></h3></th>
+					<th><h3><?php esc_html_e( 'Core Events', 'query-monitor' ); ?></h3></th>
+					<th><h3><?php esc_html_e( 'Custom Events', 'query-monitor' ); ?></h3></th>
+					<th><h3><?php esc_html_e( 'Doing Cron', 'query-monitor' ); ?></h3></th>
+					<th><h3><?php esc_html_e( 'Next Event', 'query-monitor' ); ?></h3></th>
+					<th><h3><?php esc_html_e( 'Current Time', 'query-monitor' ); ?></h3></th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td><?php echo absint( $data['total_crons'] ); ?></td>
+					<td><?php echo absint( $data['total_core_crons'] ); ?></td>
+					<td><?php echo absint( $data['total_user_crons'] ); ?></td>
+					<td><?php true === $this->doing_cron ? esc_html_e( 'Yes', 'query-monitor' ) : esc_html_e( 'No', 'query-monitor' ); ?></td>
+					<td>
+						<?php
+						echo esc_html( $data['next_event_time']['human_time'] );
+						echo '<br>';
+						echo absint( $data['next_event_time']['unix'] );
+						echo '<br>';
+						echo '<i>' . esc_html( $this->display_past_time( human_time_diff( $data['next_event_time']['unix'] ), $data['next_event_time']['unix'] ) ) . '</i>';
+						?>
+					</td>
+					<td><?php echo esc_html( gmdate( 'H:i:s' ) ); ?></td>
+				</tr>
+			</tbody>
+		</table>
+		<br>
+		<h3><b>Schedules</b></h3>
```

Comment:
Isn't `<h3>` already bold (I didn't check, just asking :-) )?

Reply:
It makes it extra bold then D:

Reply:
See also: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b